.idea/modules.xml

```diff
@@ -5,6 +5,7 @@
       <module fileurl="file://$PROJECT_DIR$/dist/plugins/DefaultPlugin/DefaultPlugin.iml" filepath="$PROJECT_DIR$/dist/plugins/DefaultPlugin/DefaultPlugin.iml" />
       <module fileurl="file://$PROJECT_DIR$/dist/plugins/PluginBase/PluginBase.iml" filepath="$PROJECT_DIR$/dist/plugins/PluginBase/PluginBase.iml" />
       <module fileurl="file://$PROJECT_DIR$/RoddyCore/RoddyCore.iml" filepath="$PROJECT_DIR$/RoddyCore/RoddyCore.iml" />
+      <module fileurl="file://$PROJECT_DIR$/../RoddyInfra/RoddyToolLib/RoddyToolLib.iml" filepath="$PROJECT_DIR$/../RoddyInfra/RoddyToolLib/RoddyToolLib.iml" />
```

Comment:
Hmm... the RoddyInfra project is upper case letter in my case. However, I don't think this will be an issue. It was this way, when I cloned it.

I think it fits in quite nicely, but we can also keep it separate.

Or do you speak about the names for the subprojects=